code/__DEFINES/antagonists.dm

```diff
@@ -52,6 +52,14 @@
 #define MAX_LEADERS_GANG		3
 #define INITIAL_DOM_ATTEMPTS	3
 
+//Shuttle elimination hijacking
+/// Does not stop elimination hijacking but itself won't elimination hijack
+#define ELIMINATION_NEUTRAL 0
+/// Needs to be present for shuttle to be elimination hijacked
+#define ELIMINATION_ENABLED 1
+/// Prevents elimination hijack same way as non-antags
+#define ELIMINATION_PREVENT 2
+
```

Comment:
There are HIJACK_NEUTRAL and other defines around 20 lines above. As this PR replaces those defines with these defines, you should delete them.

https://github.com/BeeStation/BeeStation-Hornet/blob/f6059b7d06b3380885a264e8d4842b7d58ef2fac/code/__DEFINES/antagonists.dm#L35-L38